src/queryClient.ts

```diff
@@ -1,19 +1,22 @@
+import { getReasonPhrase, StatusCodes } from 'http-status-codes';
 import { QueryClient, QueryClientProvider, useMutation } from 'react-query';
 import { ReactQueryDevtools } from 'react-query/devtools';
-import { getReasonPhrase, StatusCodes } from 'http-status-codes';
-import configureMutations from './mutations';
-import configureHooks from './hooks';
 import {
   CACHE_TIME_MILLISECONDS,
   STALE_TIME_MILLISECONDS,
 } from './config/constants';
+import configureHooks from './hooks';
+import configureMutations from './mutations';
+import configureWebSockets from './ws';
```

Comment:
Since you use `websocket` not `webSocket` everywhere else, perhaps here it's better to call it `configureWebsockets` for consistency.

Reply:
I actually changed it following your previous comment: https://github.com/graasp/graasp-query-client/pull/11#discussion_r658128973